components/Comment.js

```diff
@@ -55,6 +71,17 @@ class Comment extends React.Component {
     this.state.mode === 'edit' ? this.cancelEdit() : this.edit();
   }
 
+  handleDelete = async () => {
+    try {
+      await this.props.deleteComment(this.state.comment.id);
+      await this.props.refetch();
+      this.setState({ showDeleteModal: false });
+    } catch (err) {
+      console.error(err);
```

Comment:
Not great to hide errors

Reply:
Will be taken care of in this issue https://github.com/opencollective/opencollective/issues/2567